src/routes/location.route.js

```diff
@@ -0,0 +1,8 @@
+const express = require("express");
+const router = express.Router;
+
+const Axios = require("axios");
+
+router.get("/", (req,res) => {
+    
```

Comment:
This does nothing with the response and could hang your application, it doesn't have to do much, just a 200 status and a simple message is enough for now :)